### PR TITLE
feat: duplicate linked task in project

### PR DESCRIPTION
**Pull request by Mangesh-Khairnar**
- Copies all the linked task in the project by adding a separate button for duplicate
Screenshot 2019-10-12 at 8 12 55 PM

### CODE REVIEW

File: erpnext/projects/doctype/project/project.js [hunk 1]
```diff
@@ -80,6 +80,15 @@ frappe.ui.form.on("Project", {
 		frm.events.set_buttons(frm);
 	},
 
+	onload_post_render: function(frm) {
+		$(".menu-item-label[data-label='Duplicate']")
+			.parent('a')
+			.unbind("click")
+			.bind("click", function() {
+				frm.trigger('create_duplicate_project');
```
Comment (nabinhait) [inline erpnext/projects/doctype/project/project.js]: Function definition is `create_duplicate`?????

**Review by nabinhait — CHANGES_REQUESTED** [review]
@Mangesh-Khairnar I don't think it should copy all tasks **implicitly** on click of **standard duplicate** button. If we want to copy tasks along with project then we should add another custom button "Duplicate Project with Tasks" button.

File: erpnext/projects/doctype/project/project.js (hunk 1 above)
Comment (Mangesh-Khairnar) [inline erpnext/projects/doctype/project/project.js]:
```suggestion
				frm.trigger('create_duplicate');
```

Comment (Mangesh-Khairnar) [reply erpnext/projects/doctype/project/project.js]: Added changes

**Comment by Mangesh-Khairnar** [comment]
@nabinhait Changed to a separate button "Duplicate Project with Tasks"

**Review by nabinhait — APPROVED** [review]

**Pull request merged**